packages/jest-cli/src/__tests__/watch_filename_pattern_mode.test.js

```diff
@@ -127,6 +127,26 @@ describe('Watch mode flows', () => {
     // globalConfig is updated with the current pattern
     expect(runJestMock.mock.calls[0][0].globalConfig).toMatchSnapshot();
   });
+  it('Pressing "P" enters pattern mode (invalid regexp)', () => {
```

Comment:
Make sure to update the snapshot after renaming the test :)

Reply:
I have fixed it.
But now, I'm getting some unexpectable merge conflicts. Dont know what to do with them

Reply:
@SimenB If allow edits from maintainers is ticked, you probably have write access, can you do the rebase?

Reply:
done